src/javascript/app/ng-wig/ng-wig-toolbar.provider.spec.js

```diff
@@ -5,126 +5,128 @@ describe('provider: ngWigToolbar', function () {
         module('ngWig', (_ngWigToolbarProvider_) => {
             toolbarProvider = _ngWigToolbarProvider_;
         });
+
+        inject();
```

Comment:
interesting. did not think of it. 
basically you want to reannotate the modules 
https://github.com/angular/bower-angular-mocks/blob/master/angular-mocks.js#L3095